.claude/skills/add-cuda-kernel/skill.md

```
@@ -0,0 +1,739 @@
+    common_check=None,  # Optional: shared validation for all backends
+)
+def attention(q, k, v, backend="cutlass"):
+    if backend == "cutlass":
+        # CUTLASS implementation
+        pass
+    elif backend == "cudnn":
+        # cuDNN implementation
+        pass
+```
+
+**Pattern 3: Auto Backend Selection**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)